docs/guide/running-extern-robot-controllers.md

```diff
@@ -255,13 +159,28 @@ This variable is computed from the `SNAP_USER_COMMON` environment variable which
 Similarly, the libController will automatically check this folder and its contents to determine if it should use it to communicate with Webots.
 It is recommended that you do not override this `WEBOTS_TMPDIR` environment variable, unless you want to experiment a different mechanism.
 
+### Running a MATLAB Extern Robot Controller
+
+Matlab controllers can also be started using the launcher.
+By default, the launcher will look for the latest installed version of MATLAB in the following locations, depending on the OS:
+
+- **Windows**: C:\Program Files\MATLAB\R20XXx\bin\win64\MATLAB.exe
+- **Linux**: /usr/local/MATLAB/R20XXx/bin/matlab
+- **MacOS**: /Applications/MATLAB_R20XXx.app
+
+It is also possible to give a custom MATLAB installation path to the launcher, by providing the absolute path to the executable in the `--matlab-path` option.
+
 ## Example Usage
 
 1. Open for example the "WEBOTS\_HOME/projects/robots/softbank/nao/worlds/nao_demo.wbt" world file.
 2. If the simulation was running, stop it and revert it.
 3. Then, open the Nao node in the scene tree and change its controller field from `nao_demo` to `<extern>`.
 4. Save the simulation, restart it and run it.
-5. Open the "WEBOTS\_HOME/projects/robots/softbank/nao/controllers/nao_demo" folder from a terminal.
-6. Setup environment variables needed for a C/C++ controller as explained in the above section.
-7. Start the `nao_demo` controller manually from the terminal.
-8. You should see the Nao robot moving in the simulation, controlled by the `nao_demo` program you just started.
+5. Open a terminal and go to the "WEBOTS\_HOME/projects/robots/softbank/nao/controllers/nao_demo" folder. Note that other controllers could be started from anywhere, but in the example of the Nao, motion files are given relative to this folder. Executing the controller from another location would lead to an error in finding these files.
+6. Start the `nao_demo` controller with:
+
+  ```bash
+  $WEBOTS_HOME/webots-controller ./nao_demo  --protocol=ipc --port=1234 --robot-name=NAO
```

Comment:
Also, the `--port` and `--robot-name` options can be omitted as they are the default values.

Reply:
Yes it is correct that here no option is necessary, as these are default values, but because this is an example I wanted to show the usage of options. Maybe I can change it to tcp, so that `protocol` and `ip-address` options make sense.